CHANGELOG.md

```diff
@@ -1,7 +1,38 @@
 # Upcoming Release
 
 ## New Features:
-No changes to highlight.
+
+### New `gr.BarPlot` component! 📊
+
+Create interactive bar plots from a high-level interface with `gr.BarPlot`. 
+No need to remember matplotlib syntax anymore!
+
+Example usage:
+
+```python
+from vega_datasets import data
+
+barley = data.barley()
+
+with gr.Blocks() as demo:
+    gr.BarPlot.(
+        barley,
+        x="year",
+        y="yield",
+        color="year",
+        group="site",
+        title="Barley Yield by Year and Site",
+        group_title="",
+        tooltip=["yield", "site", "year"],
+        vertical=False
+    )
+
+demo.launch()
```

Comment:
So when I run this demo, I get this plot produced. Are there some settings that are off? 
<img width="1019" alt="image" src="https://user-images.githubusercontent.com/1778297/217909769-c9774920-08de-434f-9334-0e5af0bbaba4.png">

Reply:
Ah need to make the `year` column a str otherwise it will be interpreted as numeric which is why the x-axis is weird.

I'll just use a simpler demo for the changelog. I was trying to show off as much of the API as possible but that may be confusing.

Reply:
Fixed the typo and used a simpler demo with easier data 😄